humble/distribution.yaml

```diff
@@ -3831,7 +3831,7 @@ repositories:
       tags:
         release: release/humble/{package}/{version}
       url: https://github.com/ros2-gbp/rcutils-release.git
-      version: 5.1.1-2
+      version: 6.0.1-1
```

Comment:
This is a major bump, and it looks like the version on the Rolling branch. Was the wrong branch released from?

https://github.com/ros2/rcutils

Reply:
Oof, yeah we probably don't want a major bump. Good catch.

Reply:
Yes, this was wrong in the gbp repo.  Fixed in https://github.com/ros2-gbp/rcutils-release/pull/6 , I'll close this and do a re-release.

Reply:
Thank you!